CMakeLists.txt

```diff
@@ -15,11 +15,13 @@ include(CMakePackageConfigHelpers)
 if(EXISTS ${CMAKE_BINARY_DIR}/conanbuildinfo_multi.cmake)
     include(${CMAKE_BINARY_DIR}/conanbuildinfo_multi.cmake)
     conan_basic_setup()
+    find_package(mdspan CONFIG REQUIRED)
 elseif(EXISTS ${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
     include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
     conan_basic_setup()
+    find_package(mdspan CONFIG REQUIRED)
 else()
-    FetchContent_Declare(mdspan GIT_REPOSITORY https://github.com/kokkos/mdspan.git GIT_TAG stable)
+    FetchContent_Declare(mdspan GIT_REPOSITORY https://github.com/kokkos/mdspan.git GIT_TAG 9d1acac543053cbe6839273f550b1ece218e9696)
```

Comment:
I've pinned the build at git tag 9d1acac543053cbe6839273f550b1ece218e9696 as this is the same version of MDSpan used for the MDSpan Conan package (which is used with the Conan builds).  The latest MDSpan has removed basic_mdspan so the build is no longer compiling at head.